### ISSUE TITLE
[Bug] Generic Icon in the Programs and Features Listing

### ISSUE THREAD

**Issue by ravindUwU** [ForkIssues/TrackerWin#1114]
This is a pretty minor issue, but Fork's listing under _Programs and Features_ (`appwiz.cpl`) shows an icon for a generic application, instead of its logo <img src="https://fork.dev/images/logo.png" alt="Fork logo" width="18">.

![Fork listed in the Programs and Features window](https://user-images.githubusercontent.com/7352580/109408959-94be4780-79e2-11eb-8c05-7912c0aa135f.png)

**Comment by ravindUwU** [comment]
This no longer seems to be an issue as of `2.6.0` (possibly earlier) 🎉. #861 and #2037 should be resolved as well.

![](https://github.com/user-attachments/assets/8258a0ca-2585-44de-9c91-590d031721fd)

![](https://github.com/user-attachments/assets/1aba1c30-33d6-47cb-a818-e9f63072e284)

**Comment by DanPristupov** [comment]
@ravindUwU yes, you are right. The icon was fixed in 2.6.0. Thank you for noticing that 👍
